mcs_benchmark_data/pipelines/cycic/roberta4_cycic_submission_file_names.py

```diff
@@ -0,0 +1,13 @@
+from typing import NamedTuple
+from dataclasses import dataclass
+from dataclasses_json import LetterCase, dataclass_json
+
+from mcs_benchmark_data._benchmark_file_names import (
+    _BenchmarkFileNames,
+)
+
+
+@dataclass_json(letter_case=LetterCase.CAMEL)
+@dataclass(frozen=True)
+class Roberta4CycicSubmissionFileNames(_BenchmarkFileNames):

```

Comment:
Most of the *FileNames subclasses are going to have a few forms, like ones with inline labels and those without. You may be able to get away with having one subclass per form rather than one per benchmark or submission, then hard-coding the default file names in the pipeline (which you have to do per benchmark or submission, in any case, because it has a specialized transformer).

No need to change this PR.